draft-ietf-mls-protocol.md

```diff
@@ -2888,7 +2888,7 @@ struct {
 ~~~
 
 ~~~ pseudocode
-interim_transcript_hash_[0] = ""; /* zero-length octet string */
+confirmed_transcript_hash_[0] = ""; /* zero-length octet string */
```

Comment:
The indexing here is a bit confusing : if `n` starts at `0`, then the computation of `confirmed_transcript_hash_[n]` below contradicts this starting point, and the computation of `interim_transcript_hash_[n+1]` means there is no interim hash at `n=0`.

How about this instead : instead of `n`, use `epoch` and say explicitly that `x_hash[epoch]` is stored in the state when epoch id is `epoch` and `XHashInput[epoch]` contains values from the commit creating `epoch`.  Then swap the computations below so that for any `epoch >= 0` we compute 1) `interim_transcript_hash[epoch]` using `confirmed_transcript_hash[epoch]` and 2) `confirmed_transcript_hash[epoch+1]` using `interim_transcript_hash[epoch]` and `ConfirmedTranscriptHashInput[epoch+1]`?